<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialCommunication</class>
 <widget class="QMainWindow" name="SerialCommunication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>SerialCommunication</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/images/SpaceX.ico</normaloff>:/images/images/SpaceX.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEditReceive">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>320</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>12</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label1">
        <property name="font">
         <font>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>AIN1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAin1">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label2">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>AIN2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAin2">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label3">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>AIN3:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAin3">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label4">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>AIN4:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAin4">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label5">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>AIN5:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAin5">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label6">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>AIN6:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAin6">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>AIN7:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAin7">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label8">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>AIN8:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAin8">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label9">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>AIN9:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAin9">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTextEdit" name="textEditSend">
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>12</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="3">
       <widget class="QComboBox" name="cboxSerialParityBit">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="cboxSerialDataBits">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="btnSend">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>串口号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>波特率：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="btnOpenSerial">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>停止位：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cboxSerialBaudRate">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="cboxSerialStopBits">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="cboxSerialNum">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>10</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>校验位：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>数据位：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>轮询时间: (ms)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QSpinBox" name="spinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei</family>
          <pointsize>12</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actClearTx"/>
    <addaction name="actClearRx"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actNew">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actClearRx">
   <property name="text">
    <string>清除Rx</string>
   </property>
   <property name="toolTip">
    <string>清除接收区</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actClearTx">
   <property name="text">
    <string>清除Tx</string>
   </property>
   <property name="toolTip">
    <string>清除发送区</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
